database/migrations/2016_06_07_153443_create_smart_lives_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateSmartLivesTable extends Migration
```

Comment:
Well, even the table you are creating is the same, a database with two identical tables...